lib/sort_by_length.rb

```diff
@@ -1,7 +1,31 @@
 # A method which will return an array of the words in the string
 #  sorted by the length of the word.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: The time complexity of this method is O(n) where n is the length of the my_sentence_array, or the number of words in the my_sentence string. The time that the algorithm takes will increase linearly with the number of words passed in. 
```

Comment:
The time complexity is O(n<sup>2</sup>).  Look at the nested loops

Otherwise nice selection sort.